src/Integration/src/RabbitMQ/Steeltoe.Integration.RabbitMQ.csproj

```diff
@@ -1,7 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <Description>Steeltoe Integration RabbitMQ</Description>
+    <Description>
+      Steeltoe Integration RabbitMQ
+      **PLEASE NOTE** The public API of this package is subject to change and is not recommended for direct use outside of Steeltoe.
+    </Description>
     <TargetFrameworks>netcoreapp3.1;net5.0;</TargetFrameworks>
     <RootNamespace>Steeltoe.Integration.RabbitMQ</RootNamespace>
     <PackageTags>Integration, ASPNET Core, Spring, Spring Cloud, RabbitMQ</PackageTags>
```

Comment:
Why are the package tags Spring and Spring Cloud?

Reply:
I didn't add them but I can see how they would help someone looking for nugets in this space. For example typing BlockingQueue in my IDE suggests adding a IKVM project - helps discovery.